### PR TITLE
Fixed user facing and comment typos

### PR DESCRIPTION
**Pull request by sean-gilliam**
This PR fixes a number of typos in the text displayed to the user. It also fixes a number of typos in the comments.

There's at least a handful of typos in function and variable names, but didn't want to mix concerns with this PR.

Used the extension **Code Spell Checker** for the spell check. Found here https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

### CODE REVIEW

File: code/act_info.c [hunk 1]
```diff
@@ -1822,7 +1822,7 @@ void do_look(CHAR_DATA *ch, char *argument)
 
 		if (is_affected_room(ch->in_room, gsn_glaciate) && ch->in_room->sector_type == SECT_ICE)
 		{
-			sprintf(buf, "%sThe water here has congealed into a thick glacial icefloe!%s\n\r",
+			sprintf(buf, "%sThe water here has congealed into a thick glacial iceflow!%s\n\r",
```
Comment (Psypher9) [inline code/act_info.c:1825]: I think this one was correct if it was `ice floe`
```suggestion
			sprintf(buf, "%sThe water here has congealed into a thick glacial ice floe!%s\n\r",
```

File: code/olc.c [hunk 2]
```diff
@@ -1475,7 +1475,7 @@ void do_resets(CHAR_DATA *ch, char *argument)
```
Comment (Psypher9) [inline code/olc.c:1475]: I think the original lines look like they're from Spanish and so these should actually translate from `... no existe` to `... does not exist`

**Review by Psypher9 — CHANGES_REQUESTED** [review]

File: code/olc.c [hunk 3]
```diff
@@ -1475,7 +1475,7 @@ void do_resets(CHAR_DATA *ch, char *argument)
 
 				if (get_mob_index(is_number(arg3) ? atoi(arg3) : 1) == nullptr)
 				{
-					send_to_char("Mob no existe.\n\r", ch);
+					send_to_char("Mob no exists.\n\r", ch);
```
Comment (Psypher9) [inline code/olc.c:1478]: This was meant to be connected with the comment above
```suggestion
					send_to_char("Mob does not exist.\n\r", ch);
```

File: code/olc.c [hunk 4]
```diff
@@ -1511,7 +1511,7 @@ void do_resets(CHAR_DATA *ch, char *argument)
 
 					if (get_obj_index(atoi(arg3)) == nullptr)
 					{
-						send_to_char("Vnum no existe.\n\r", ch);
+						send_to_char("Vnum no exists.\n\r", ch);
```
Comment (Psypher9) [inline code/olc.c:1514]: This one as well:
```suggestion
						send_to_char("Vnum does not exist.\n\r", ch);
```

File: code/olc.c [hunk 5]
```diff
@@ -1532,7 +1532,7 @@ void do_resets(CHAR_DATA *ch, char *argument)
...
```
Comment (Psypher9) [inline code/olc.c:1535]: And finally this:
```suggestion
						send_to_char("Vnum does not exist.\n\r", ch);
```

**Review by Psypher9 — CHANGES_REQUESTED** [review]

**Comment by sean-gilliam** [comment]
Changed wording based on @Psypher9 suggestions.

File: code/handler.c [hunk 6]
```diff
@@ -3188,7 +3188,7 @@ char *extra_bit_name(long extra_flags[])
 		strcat(buf, " anti-lawful");
 
 	if (IS_SET(extra_flags, ITEM_ANTI_NEUT))
-		strcat(buf, " anti-neutrale");
+		strcat(buf, " anti-neutrale");	// TODO: should this be anti-neutral
```
Comment (rezalas) [inline code/handler.c:3191]: It really should...

**Comment by sean-gilliam** [comment]
Fixed todo based on @rezalas input.

**Review by rezalas — APPROVED** [review]
I firmly believe that nobody working on this before 2012 could spell.

**Pull request merged**